content/developer/writing_widgets/widget-definition.md

```diff
@@ -140,8 +140,13 @@ fetchUrl: '[manager]/executions?is_system_workflow=false[params]'
     * `[params:param_name1(,param_name2)]` is replaced with "&paramName1:paramValue1" in the URL.
      Please note that this can be used both to selectively pick pagination parameter as well as custom parameters (see [fetchParams function](#fetchparams-widget-toolbox)).
       This mode is **exclusive** - parameters not specified explicitly are skipped.
-      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include all pagination parameters (`_size`, `_offset`, `_sort`) instead of specifying explicitly  each of the three.
-
+      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include [pagination parameters](https://docs.cloudify.co/api/v3.1/#pagination) (`_size`, `_offset`), [sorting parameter](https://docs.cloudify.co/api/v3.1/#sorting) (`_sort`) and search parameter (`_search`) instead of specifying explicitly each of the four. It means `fetchUrl: '[manager]/agents?[params:gridParams]'` is equivalent for 
+      `fetchUrl: '[manager]/agents?[params:_offset,_size,_sort,_search]'`.
+      
+      Useful resources: 
+        * [`DataTable` documentation](https://docs.cloudify.co/ui-components/2.3.0/?path=/docs/home--page),
```

Comment:
I think the comment from @Gelio - https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1713#discussion_r576026451 - is not addressed. It was about pointing out that it is not good to reference a link which is dynamic (version will for sure change in the future). Changing the link to currently latest is not a solution.
The best would be of course to reuse ui_components_link parameter as @Gelio suggested, but let's leave it for now. 

Why didn't you add reference to widget components API page (`developer/writing_widgets/widgets-components`) as I suggested at the beginning?
It is not perfect, because you have to click further to get to UI Components documentation, but it is much better to avoid problems in the future with referencing the version. Another argument for not using direct reference to `https://docs.cloudify.co/ui-components` is that we already reference widget components API page in few places.

Reply:
I make mistake, should be:
```
* [widget definition]({{< relref "developer/writing_widgets/widgets-components" >}}) - to know how to develop widgets,
```

Reply:
fixed